flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2499,9 +2594,30 @@ class FMCMainDisplay extends BaseAirliners {
 
     //TODO: fix this functionality
     trySetPreSelectedDescentSpeed(s) {
+        const isNextPhase = this.currentFlightPhase === FmgcFlightPhases.CRUISE;
```

Comment:
Doesn't look like this should be editable according to FCOM.
<img width="716" alt="image" src="https://user-images.githubusercontent.com/9995998/116978230-8a3c6900-ad17-11eb-8778-35566f4f464f.png">

Reply:
This is possible in FMS release 2.

Reply:
Disregard, I was talking about changing the managed speed in 3L, not the selected speed in 4L